src/core/router/net_db/impl.cc

```diff
@@ -668,7 +671,8 @@ void NetDb::HandleDatabaseLookupMsg(
   }
   std::uint16_t num_excluded = bufbe16toh(excluded);
   excluded += 2;
-  if (num_excluded > static_cast<std::uint16_t>(NetDbSize::MaxExcludedPeers)) {
+  if (num_excluded >
+      static_cast<std::uint16_t>(NetDbSize::MaxExcludedPeers)) {
     LogPrint(eLogWarn,
         "NetDb: number of excluded peers", num_excluded, " exceeds 512");
```

Comment:
The issue wasn't indentation but rather the log message. https://github.com/monero-project/kovri/pull/444#discussion_r87059962